docs/release-notes/release-notes-1.7.2.md

```diff
@@ -0,0 +1,29 @@
+# dcrd v1.7.2
+
+This is a patch release of dcrd to resolve a rare and hard to hit case when
+optional indexing is enabled.
+
+## Changelog
+
+This patch release consists of 4 commits from 2 contributors which total to 11
+files changed, 158 additional lines of code, and 15 deleted lines of code.
+
+All commits since the last release may be viewed on GitHub
+[here](https://github.com/decred/dcrd/compare/release-v1.7.1...release-v1.7.2).
```

Comment:
I actually included the commit that came in via the updated `blockchain` module (the [release-v1.7] main: Use backported blockchain updates) which doesn't show up directly on the release branch itself since it had to be backported separately to the last release version of the module.

I went back and forth a bit on whether or not to include that in the statistics since I know it means the release branch link itself won't match up, but it is technically a part of it.

Reply:
Perhaps I should just include the module compare link as well.  Such as:

All commits since the last release may be viewed on GitHub
[here](https://github.com/decred/dcrd/compare/release-v1.7.1...release-v1.7.2) and [here](https://github.com/decred/dcrd/compare/blockchain/v4.0.0...blockchain/v4.0.1).

Reply:
Updated and included the relevant commit in the breakout.

Reply:
Makes more sense, thank you!